modules/javafx.web/src/main/java/com/sun/webkit/graphics/WCRenderQueue.java

```diff
@@ -171,6 +171,15 @@ public boolean isOpaque() {
         return opaque;
     }
 
+    /*is called from native*/
+    private synchronized boolean strokeContains(WCPath path, float x, float y) {
```

Comment:
We usually prefix methods that are called form native WebKit code with `fwk`. Arun can comment as to whether it is needed here.